app/api/routers/breeze_buddy/telephony/answer/handlers.py

```
@@ -463,11 +500,60 @@ async def _create_inbound_lead_in_answer_handler(
             ),
             call_direction=CallDirection.INBOUND,
         )
+        if not lead:
+            logger.error(
+                f"[Answer] Failed to create inbound lead for call_id {call_id}"
+            )
+            return None
         logger.info(f"[Answer] Created inbound lead {lead_id} for call_id {call_id}")
+        return lead
     except Exception as e:
```

Ruff (0.16.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)